doc/api.rst

```diff
@@ -30,7 +30,7 @@ Utils (:py:mod:`mne_bids.utils`):
 
 Datasets (:py:mod:`mne_bids.datasets`):
 
-.. currentmodule:: mne.datasets
+.. currentmodule:: mne_bids.datasets
 
```

Comment:
This was the bug killing our current circle_ci ... I have no clue why this didn't show up in the PR that @teonbrooks merged ... back then, the CIs were green ...

cc @jasmainak

Reply:
thanks for the catch!